resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -198,6 +199,12 @@ def get_proxies(self):
     def has_inputstream_adaptive_installed(self):
         return xbmc.getCondVisibility('System.HasAddon("{0}")'.format('inputstream.adaptive')) == 1
 
+    def has_white_icons(self):
+        return xbmc.getCondVisibility('System.HasAddon("resource.images.studios.white")') == 1
+
+    def has_coloured_icons(self):
```

Comment:
so this will work when you manually install the color addon? Otherwise its white?

Reply:
nvm saw the comment

Reply:
@mediaminister also created the coloured icons, but they have been reverted as they were in the wrong format. We'll have to wait for them to be resubmitted and released :-/